Owl/app/src/main/java/com/example/owl/ui/NavGraph.kt

```diff
@@ -16,37 +16,70 @@
 
 package com.example.owl.ui
 
-import android.os.Parcelable
-import androidx.compose.runtime.Immutable
-import com.example.owl.ui.utils.Navigator
-import kotlinx.android.parcel.Parcelize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.navigation.NavHostController
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.navArgument
+import androidx.navigation.compose.navigate
+import androidx.navigation.compose.rememberNavController
+import com.example.owl.ui.MainDestinations.COURSE_DETAIL_ID_KEY
+import com.example.owl.ui.course.CourseDetails
+import com.example.owl.ui.courses.Courses
+import com.example.owl.ui.onboarding.Onboarding
 
 /**
- * Models the screens in the app and any arguments they require.
+ * Destinations used in the main screen ([OwlApp]).
```

Comment:
This seems misleading – Onboarding isn't on the main screen

Reply:
Fixed but let's revisit naming when we include the bottom bar, I'm not sure if it's a subgraph or we'll just add destinations here.